onboarding/lambdapy.md

```diff
@@ -0,0 +1,29 @@
+## Installation
+
+Rollbar needs to be installed like any other dependency that you intend to use by including it in your deployment package ( [Instructions here](http://docs.aws.amazon.com/lambda/latest/dg/lambda-python-how-to-create-deployment-package.html))
+
+This basically amounts to running
+
+```
+pip install rollbar
+```
+
+## Sending a message to Rollbar
+
+The main change that is required to get Rollbar to work is to either use the `blocking` handler
```

Comment:
This change is relative to the non-lambda instructions, which isn't meaningful to new users in onboarding since this is their first exposure to Rollbar.

I think these instructions should include:

- the rollbar.init call to run (which I think no longer requires the `blocking` handler after @rokob's latest changes?)
- an example of how to use the decorator. (the one you have on lines 23-26 looks great)